lib/src/specs/image/image_util.dart

```diff
@@ -16,21 +16,29 @@ class ImageUtility extends SpecUtility<ImageSpecAttribute> {
     ColorDto? color,
     ImageRepeat? repeat,
     BoxFit? fit,
+    AlignmentGeometry? alignment,
+    Rect? centerSlice,
+    BlendMode? blendMode,
+    FilterQuality? filterQuality,
   }) {
     return ImageSpecAttribute(
       width: width,
       height: height,
       color: color,
       repeat: repeat,
       fit: fit,
+      alignment: alignment,
+      centerSlice: centerSlice,
+      colorBlendMode: blendMode,
+      filterQuality: filterQuality,
     );
   }
 
   ColorUtility<ImageSpecAttribute> get color {
     return ColorUtility((color) => _only(color: color));
   }
 
-  ImageRepeatUtility<ImageSpecAttribute> get repeat {
+  ImageRepeatUtility<ImageSpecAttribute> get imageRepeat {
```

Comment:
This would still be `image.imageRepeat()`, what I suggest is what you had before, and try to remove redundant words. so we would keep `repeat,` but add a callable method to return `ImageRepeat.repeat`

Reply:
Sorry, my bad 😅